onnx_coreml/_operators.py

```diff
@@ -1522,8 +1522,10 @@ def _convert_upsample(builder, node, graph, err):  # type: (NeuralNetworkBuilder
         height_scale = int(scales[2])
         width_scale = int(scales[3])
     else:
-        height_scale = int(node.attrs.get('height_scale', 1))
```

Comment:
can you also keep the old behavior intact? That is, if the keys are present use them otherwise use scales. 
Also if you can add a uni test it would be great. Thanks!

Reply:
done?